README.md

```diff
@@ -386,6 +387,12 @@ __Alternatives:__
 git whatchanged --since='2 weeks ago'
 ```
 
+## Which files have changed most since two weeks?
+```sh
+git log -M -C --name-only --since="2 weeks ago" --all --format='format:' "$@" | sort | grep -v '^
+ | uniq -c | sort -n | awk 'BEGIN {print "count\tfile"} {print <!-- @doxie.inject start --> "\t" <!-- @doxie.inject end -->}'
```

Comment:
Yikes!

Reply:
This happened when I sent my changes   :worried:

the original alias is:

**git log -M -C --name-only --since="midnight" --all --format='format:' "$@" | sort | grep -v '^$' | uniq -c | sort -n | awk 'BEGIN {print "count\tfile"} {print $1 "\t" $2}'**

Reply:
Do i need modify the readme file?

Reply:
Need to check doxie